node/src/node_configurator/node_configurator_standard.rs

```diff
@@ -1109,7 +1178,7 @@ mod tests {
 
     #[test]
     #[should_panic(
-        expected = "You need to define data-directory to define config file with naked directory 'dirname/config.toml'."
+        expected = "You need to define data-directory to define config file with naked directory config/config.toml."
```

Comment:
Suggested error message: "If the config file is given with a naked relative path (config/config.toml), the data directory must be given to serve as the root for the config-file path."